src/libs/dataUseTranslation.js

```diff
@@ -372,7 +372,7 @@ export const DataUseTranslation = {
       if (!isNil(value) && value) {
         if (key === 'diseaseRestrictions') {
           let resolvedLabels = [];
-          if (!value.isEmpty && !isNil(value[0].label)) {
+          if (!isNil(value[0]) && !isNil(value[0].label)) {
```

Comment:
nit: I prefer to use lodash `head` method to get the first element of an array - it's cleaner (and more functional) than referencing `[0]` directly.